buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -83,6 +83,19 @@ class BuildPlugin implements Plugin<Project> {
 
     /** Adds additional manifest info to jars */
     static void configureJarManifest(Project project) {
+        project.scminfo {
+            def dotgit = "$project.rootProject.projectDir/.git/"
```

Comment:
All of these comments are addressed by 95dd9c13ad77bde2e5ad5f3ef4e7669f0be8e51d.